<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChequePrint</class>
 <widget class="QMainWindow" name="ChequePrint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>446</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>994</width>
    <height>446</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>994</width>
    <height>446</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>چاپ چک</string>
  </property>
  <property name="locale">
   <locale language="Persian" country="Iran"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>مبلغ چک: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>150</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>در وجه: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>230</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>شناسه ملی (کد ملی):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PrintButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>310</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>چاپ</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="TarixEdit">
    <property name="geometry">
     <rect>
      <x>742</x>
      <y>20</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>100</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>مبلغ چک به حروف: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>190</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>بابت: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DarvachEdit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>431</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>تاریخ چک: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="BabetEdit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>190</y>
      <width>431</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="hlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>811</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>مبلغ چک به حروف</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MablaghEdit">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="KodemelliEdit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>230</y>
      <width>231</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>11</number>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>310</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>خروج</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>994</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>TarixEdit</tabstop>
  <tabstop>MablaghEdit</tabstop>
  <tabstop>DarvachEdit</tabstop>
  <tabstop>BabetEdit</tabstop>
  <tabstop>KodemelliEdit</tabstop>
  <tabstop>PrintButton</tabstop>
  <tabstop>ExitButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
